peerconnection.go

```diff
@@ -1947,8 +1947,14 @@ func (pc *PeerConnection) GetStats() StatsReport {
 		DataChannelsOpened:    dataChannelsOpened,
 		DataChannelsRequested: dataChannelsRequested,
 	}
+
 	pc.mu.Unlock()
 
+	senders := pc.GetSenders()
```

Comment:
It would be better to include both `Sender` and `Receiver` stats in this PR. Since they can be computed with a single pass over `pc.rtpTransceivers`

Reply:
alright cool.